packages/@roots/wordpress-externals-webpack-plugin/src/index.ts

```diff
@@ -19,9 +19,14 @@ export class Plugin {
 
   /**
    * Class constructor
+   *
+   * @todo less lazy typings (see ./externals)
    */
   constructor() {
-    this.externals = new ExternalsPlugin('window', externals)
+    this.externals = new ExternalsPlugin(
+      'window',
+      externals as any,
```

Comment:
Related to same typing issue as found in packages/@roots/wordpress-externals-webpack-plugin/src/externals.ts